react/SelectBox/SelectBox.jsx

```diff
@@ -109,6 +113,34 @@ const CheckboxOption = ({ ...props }) => <Option {...props} withCheckbox />
 
 CheckboxOption.propTypes = {}
 
+const ActionsOption = ({ actions, ...props }) => (
+  <Option {...props}>
+    {props.children}
+    <span className={styles['select-option__actions']}>
+      {actions.map((action, index) => (
+        <Icon
+          key={index}
+          icon={action.icon}
+          color={props.isFocused ? palette['coolGrey'] : palette['silver']}
+          className="u-ph-half"
+          onClick={e => {
+            e.stopPropagation()
+            action.handler(props)
+          }}
+        />
+      ))}
+    </span>
+  </Option>
+)
+
+ActionsOption.propTypes = {
+  actions: PropTypes.objectOf(PropTypes.func)
```

Comment:
The propTypes is no longer OK.

PropTypes.shape({
  icon: PropTypes.string,
  handler: PropTypes.func
})